src/config/default.ts

```diff
@@ -26,10 +26,15 @@ const defaultConfig: KarhuConfig = {
   colors,
   outputFormat: process.env.KARHU_JSON ? 'json' : 'text',
   formatters: {
-    text: (toLog: any[], logLevel: string, context: string, config: KarhuConfig, colorStart: string, colorEnd: string, transport: KarhuTransport) =>
-      `[${(transport.formatNow || config.formatNow)(config)}] ${colorStart}${logLevel} ${context} ${toLog.map(val => karhuInspect(val)).join(' ')}${colorEnd}`,
+    text: (toLog: any[], logLevel: string, context: string, config: KarhuConfig, colorStart: string, colorEnd: string, transport: KarhuTransport) => {
+      const contextualData = config.getContextualData && config.getContextualData()
```

Comment:
Could use optional calls here now that we have optional chaining
```suggestion
      const contextualData = config.getContextualData?.()
```